<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtVerifyClass</class>
 <widget class="QMainWindow" name="qtVerifyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtVerify</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>总检数据</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>热量表</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>阀门状态</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>数据采集</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>操作步骤</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_Para">
         <property name="text">
          <string>Para</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string> Air-Out</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_Start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_Save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_Exit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystemSet">
    <property name="title">
     <string>SystemSet</string>
    </property>
    <addaction name="action_spset"/>
    <addaction name="action_database"/>
    <addaction name="action_sql"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>VerifyItem</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menuDataQuery">
    <property name="title">
     <string>DataQuery</string>
    </property>
    <addaction name="action_dataquery"/>
   </widget>
   <addaction name="menuSystemSet"/>
   <addaction name="menu"/>
   <addaction name="menuDataQuery"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_spset">
   <property name="text">
    <string>spset</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>流量传感器-质量法</string>
   </property>
  </action>
  <action name="action_dataquery">
   <property name="text">
    <string>流量传感器-质量法</string>
   </property>
  </action>
  <action name="action_database">
   <property name="text">
    <string>database</string>
   </property>
  </action>
  <action name="action_sql">
   <property name="text">
    <string>sql</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Exit</tabstop>
 </tabstops>
 <resources>
  <include location="../qtverify.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>qtVerifyClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
